public/app/core/context/GrafanaContext.ts

```diff
@@ -26,3 +27,18 @@ export function useGrafana(): GrafanaContextType {
   }
   return context;
 }
+
+// Implementation of useReturnToPrevious that's made available through
+// @grafana/runtime
+export function useReturnToPreviousInternal() {
+  const { chrome } = useGrafana();
+
+  const setReturnToPrevious = useCallback(
+    (rtp: ReturnToPreviousProps) => {
+      chrome.setReturnToPrevious(rtp);
+    },
+    [chrome]
+  );
+
+  return setReturnToPrevious;
```

Comment:
can't this just be:
```
  return chrome.setReturnToPrevious;
```
?

Reply:
unless i'm missing something obvious, maybe @joshhunt can chime in 😅

Reply:
yup, you're right!